classes/Twig/Block/UpgradeButtonBlock.php

```diff
@@ -179,7 +185,9 @@ public function render()
             'downloadPath' => $this->downloadPath . DIRECTORY_SEPARATOR,
             'directoryVersionNumber' => $this->config->get('directory.version_num'),
             'manualMode' => $this->manualMode,
-            'phpVersion' => PHP_VERSION,
+            'phpVersion' => substr(PHP_VERSION, 0, 6),
```

Comment:
Why the substr? I don't get it :thinking:

Reply:
indeed, I'm not sure what was intended 🤔 

(maybe there was some confusion when you made advice about using substr, you were talking about somewhere else in the code, it's useless here since it is solely for display)

fixing it 👍